internal/rukpak/source/containers_image.go

```diff
@@ -217,7 +241,7 @@ func (i *ContainersImageRegistry) unpackImage(ctx context.Context, unpackPath st
 		return fmt.Errorf("error creating image source: %w", err)
 	}
 
-	if err := os.MkdirAll(unpackPath, 0755); err != nil {
+	if err := os.MkdirAll(unpackPath, 0700); err != nil {
```

Comment:
Why the restrictive mode?

Reply:
Also noting `setReadOnlyRecursive`... these only set the owner's bits, not group/other.

Reply:
It is the same mode as we are setting in ` applyLayerFilter()`

Reply:
Just trying to be consistent every where. These files are always written and read just by us, so these permissions should be fine. The cache dir is in an emptyDir mount that starts out empty every time a pod starts, so we should never need to be reading files written by another UID.